config.py

```diff
@@ -52,6 +52,8 @@ class Config(object):
 
     SHARED_EMAIL_KEY = None
     INVITE_EMAIL_SALT = 'SupplierInviteEmail'
+    RESET_PASSWORD_SALT = 'ResetPasswordSalt'
+    BUYER_CREATION_TOKEN_SALT = 'BuyerCreation'
```

Comment:
So you'll have to ensure salt is the same as buyer front end (where the reset password emails are also sent from)?